CMakeLists.txt

```diff
@@ -293,6 +294,9 @@ find_package(JPEG REQUIRED)
 find_package(PNG REQUIRED)
 find_package(TIFF REQUIRED)
 find_package(ZLIB REQUIRED)
+if (WITH_SYSTEM_KLT)
+  find_package (KLT REQUIRED)
+endif(WITH_SYSTEM_KLT)
```

Comment:
Please don't repeat the condition in `else`s and `endif`s - use the same style as in the rest of the file.